actions/workflows/st2_e2e_tests.yaml

```diff
@@ -128,7 +128,7 @@ workflows:
                     cmd: st2 run core.remote hosts=<% $.host %> hostname
 
 
-    test_quickstart:
+    test_quickstart_wf:
```

Comment:
Sounds like workflow name collision with the subtask is somehow related, maybe started happening under this specific environment when we started to run tasks in parallel.

Or it's random Mistral bug.

Will go ahead and merge it to see in prod.